lib/acts_as_paranoid/associations.rb

```diff
@@ -31,7 +31,7 @@ def #{target}_with_unscoped(*args)
                 association = association(:#{target})
                 return nil if association.options[:polymorphic] && association.klass.nil?
                 return #{target}_without_unscoped(*args) unless association.klass.paranoid?
-                association.klass.with_deleted.scoping { #{target}_without_unscoped(*args) }
+                association.klass.with_deleted.scoping { association.klass.unscoped { #{target}_without_unscoped(*args) } }
```

Comment:
I'm going to check if this doesn't have any unwanted side effects.

Reply:
I tried to explain here why I made that change, if it helps https://github.com/ldstudios/acts_as_paranoid/pull/2

Reply:
Ok, I created some explicit tests for this and it looks like everything is fine :+1:. I'll merge this in and then create another PR to add those tests.